src/encoded/static/components/news.js

```diff
@@ -31,10 +31,12 @@ NewsPreviewItem.propTypes = {
 // Display a list of news preview items from search results.
 const NewsPreviews = (props) => {
     const { items } = props;
-    if (items && items.length) {
+    const newsItems = items && items.length ? [...items] : [];
+    newsItems.sort((item1, item2) => new Date(item2.date_created) - new Date(item1.date_created));
+    if (newsItems.length > 0) {
```

Comment:
Might as well move this sort inside the following `if` so we don’t have to call sort on a potentially zero-length array.